framework/src/modules/http_api/schema/swagger.yml

```diff
@@ -945,60 +945,60 @@ parameters:
   publicKey:
     name: publicKey
     in: query
-    description: Public key to query
+    description: Public key to query.
     type: string
     format: publicKey
   username:
     name: username
     in: query
-    description: Delegate username to query
+    description: Delegate username to query.
     type: string
     format: username
     minLength: 1
     maxLength: 20
   blockId:
     name: blockId
     in: query
-    description: Block id to query
+    description: Block ID to query.
     type: string
-    format: id
+    format: ID
```

Comment:
Do not change anything else than the descriptions

Reply:
OK